deps/UsdImagingLite/pxr/usdImaging/usdImagingLite/engine.cpp

```diff
@@ -165,100 +91,80 @@ bool UsdImagingLiteEngine::SetRendererAov(TfToken const &id)
 
 bool UsdImagingLiteEngine::GetRendererAov(TfToken const &id, void *buf)
 {
-    SdfPath renderBufferId = _taskDataDelegate->GetDelegateID().AppendElementString("aov_" + id.GetString());
+    SdfPath renderBufferId = _renderDataDelegate->GetDelegateID().AppendElementString("aov_" + id.GetString());
     HdRenderBuffer *rBuf = static_cast<HdRenderBuffer*>(_renderIndex->GetBprim(HdPrimTypeTokens->renderBuffer, renderBufferId));
     void *data = rBuf->Map();
     memcpy(buf, data, rBuf->GetWidth() * rBuf->GetHeight() * HdDataSizeOfFormat(rBuf->GetFormat()));
     rBuf->Unmap();
     return true;
 }
 
-UsdImagingGLRendererSettingsList UsdImagingLiteEngine::GetRendererSettingsList() const
```

Comment:
Why did you completely remove this method?

Reply:
We don't use it anywhere, also it is available in UsdImagingGLEngine.